charts/traefik/values.common.yaml.gotmpl

```diff
@@ -1,3 +1,6 @@
+additionalArguments:
+  - "--api.insecure=true"
```

Comment:
is that the traefiki API right? I think there is a way to protect it. At least there was in traefik v2